snap/snapcraft.yaml

```diff
@@ -67,4 +68,4 @@ parts:
       # explicitly added to stage.  Also forcibly update the pip embedded in
       # virtualenv to get the lsb_release fix that was released with pip 10.
       cp $SNAPCRAFT_PART_INSTALL/bin/pip* bin/
-      pip download pip -d lib/python*/site-packages/virtualenv_support/
+      python3 -m pip download pip -d lib/python*/site-packages/virtualenv_support/
```

Comment:
Without this change the following error is raised when trying to build the snap.

```
Priming charm-tools 
Traceback (most recent call last):
  File "/root/parts/charm-tools/install/bin/pip", line 7, in <module>
    from pip._internal.cli.main import main
ModuleNotFoundError: No module named 'pip._internal.cli.main'
Failed to run 'override-prime': Exit code was 1.```